js/common/frozen_object.go

```diff
@@ -0,0 +1,69 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/dop251/goja"
+)
+
+// FreezeObject replicates the JavaScript Object.freeze function.
+func FreezeObject(rt *goja.Runtime, obj goja.Value) error {
+	global := rt.GlobalObject().Get("Object").ToObject(rt)
+	freeze, ok := goja.AssertFunction(global.Get("freeze"))
+	if !ok {
+		panic(fmt.Errorf("failed to get the Object.freeze function from the runtime"))
```

Comment:
:see_no_evil: It's a 100% nitpick, but we don't need here formatting (`fmt.Errorf`) and maybe even an error when panicing

Reply:
You're right. I'm sorry, this is a very basic error on my side :man_facepalming:

Reply:
No worries at all, it's not a big deal :hugs: